helpdesk_ticket_customer_response/models/helpdesk_ticket_team.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Antoni Marroig(APSL-Nagarro)<amarroig@apsl.net>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HelpdeskTicketTeam(models.Model):
+    _inherit = "helpdesk.ticket.team"
+
+    change_customer_reponse_stage_tickets = fields.Boolean(
+        string="Automatic change stage of tickets answered by customer",
+        default=False,
+    )
```

Comment:
```suggestion
autoupdate_ticket_stage = fields.Boolean(
        string="Auto Update Ticket Stage", help="Update ticket stage when a new message is registered by the partner."
        default=False,
    )
```

I think would be better adding the context on 'help' parameter and get a shorter field name. What you think?